src/lib/react/useRoomConnection.ts

```diff
@@ -417,7 +417,7 @@ export default function useRoomConnection(
             },
         },
         components: {
-            VideoView,
+            VideoView: (props) => React.createElement(VideoView, Object.assign({}, props, { roomConnection })),
```

Comment:
Here Id actually provide a `onResize` property which takes care of communicating with the roomConnection.